iceoryx_hoofs/test/moduletests/test_relative_pointer.cpp

```diff
@@ -308,138 +233,120 @@ TYPED_TEST(relativeptrtests, AssignmentOperatorTests)
     }
 
     {
-        auto offset = ShmSize + 1;
-        void* adr = static_cast<uint8_t*>(memMap1.getMappedAddress()) + offset;
+        auto offset = SHARED_MEMORY_SIZE + 1;
+        void* adr = static_cast<uint8_t*>(this->memoryPartition[1]) + offset;
         RelativePointer<TypeParam> rp;
         rp = adr;
         EXPECT_NE(rp, nullptr);
     }
 }
 
-TYPED_TEST(relativeptrtests, IdAndOffset)
+TYPED_TEST(base_relative_ptr_test, IdAndOffset)
 {
-    MemMap memMap(this->m_fileDescriptor);
-    void* basePtr1 = memMap.getMappedAddress();
+    void* basePtr1 = this->memoryPartition[0];
 
-    RelativePointer<TypeParam> rp1(memMap.getMappedAddress(), 1);
-    EXPECT_EQ(rp1.registerPtr(1, memMap.getMappedAddress()), true);
+    RelativePointer<TypeParam> rp1(this->memoryPartition[0], 1);
+    EXPECT_EQ(rp1.registerPtr(1, this->memoryPartition[0]), true);
     EXPECT_EQ(rp1.getOffset(), reinterpret_cast<std::ptrdiff_t>(basePtr1));
     EXPECT_EQ(rp1.getId(), 1);
 
-    int offset = ShmSize / 2;
-    auto offsetAddr1 = reinterpret_cast<TypeParam*>(static_cast<uint8_t*>(memMap.getMappedAddress()) + offset);
+    int offset = SHARED_MEMORY_SIZE / 2;
+    auto offsetAddr1 = reinterpret_cast<TypeParam*>(this->memoryPartition[0] + offset);
```

Comment:
The name is misleading. This is an absolute address to the offset. Could you rename it to `addressAtOffset`